spring-ai-docs/src/main/antora/modules/ROOT/pages/api/mcp/mcp-client-boot-starter-docs.adoc

```diff
@@ -217,7 +217,7 @@ spring:
               url: http://localhost:8080
             server2:
               url: http://otherserver:8081
-              endpoint: /custom-sse
```

Comment:
Hi @cattaozi, Thanks for the PR. But, this example is to show how one can customise the endpoint. The default is already "/mcp" and hence, we can come up with some other name than "/mcp" - something like "custom-endpoint" if having "sse" is confusing.